workflow/qa-steps.md

```diff
@@ -0,0 +1,35 @@
+# QA steps
+
+In this section you will find a list of steps to take when reviewing PRs, in order to catch unforeseen regressions.
+
+## Adding content
+
+- If adding a new page to a site, check that it has been added to the navigation, sitemap and breadcrumbs (where applicable)
```

Comment:
It's a tad pedantic, but you should probably be consistent about whether list items have full stops at the end or not.